CONTRIBUTING.md

```diff
@@ -79,4 +114,45 @@ ensuring they pass
 
 ## Releasing
 
-TODO: Define release workflow once we have a PyPI publishing in place
+To create a tag and release, follow the instructions in this section.
+
+### Update the version, changelog, and notices
+
+1. Create a new branch for the version bump.
+
+1. Based on the unreleased content, determine the new version number and update the version in `version.py`
+
+1. Review the git log and ensure the changelog contains all relevant recent changes with references to GitHub issues or PRs, if possible.
+
+1. Review the changes since the last tag, and if the dependencies have changed revise the notices file to correctly capture the added dependencies and their licenses / copyrights.
+
+1. Before creating a release, ensure that all documentation that needs to be written has been written by TW, approved by PO/Engineer, and pushed to the forward-facing documentation.
+
+1. Commit these changes to the branch. Bump version to x.y.z is an acceptable commit message and open a PR for review.
+
+### Add a git tag
+
+1. Once your changes have been reviewed and merged into master, tag the version using `git tag -s v0.1.1`. Note this requires you to be able to sign releases. Consult the github documentation on signing commits on how to set this up vx.y.z is an acceptable tag message.
+1. Push the tag: `git push vx.y.z` (or `git push origin vx.y.z` if you are working from your local machine).
+
+### Packing
+
+Currently, packing the client into an executable is a manual process. For Linux and Windows, you must pack the client using the different VMs we have available to us. For macOS, you will need to use your local machine.
+
+#### Windows and Linux (TBD)
+
+1. Copy the updated repository from your local machine to the Windows VM
+2. Run `pyinstaller --onefile pkg_bin/conjur-cli`. The output should appear in the `dist` folder. 
+3. Once an executable has been created, place executable under `<platform>/conjur-cli` format detailed below and archieve the file in a tar.gz format and push the contents to the release
+
+#### macOS (TBD)
+
+1. Run `./bin/build_binary` to build and pack the CLI for you. The output should appear in the `dist` folder.
+2. Once an executable has been created, place executable under `<platform>/conjur-cli` format detailed below and archieve the file in a tar.gz format and push the contents to the release
```

Comment:
place it

Reply:
Updating to 
"Add the archived file as an asset in the release"